streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamTask.java

```diff
@@ -154,15 +153,15 @@ public StreamTask(final TaskId id,
         for (final String terminalNodeName : topology.terminalNodes()) {
             e2eLatencySensors.put(
                 terminalNodeName,
-                TaskMetrics.e2ELatencySensor(threadId, taskId, terminalNodeName, RecordingLevel.INFO, streamsMetrics)
+                TaskMetrics.e2ELatencySensor(threadId, taskId, terminalNodeName, streamsMetrics)
```

Comment:
This is not introduced by this PR, but I'm wondering why we record the `maybeRecordE2ELatency` twice, once before the record is being processed and once when it reaches the sink node?

Reply:
NVM, I realized it is at the per-node metrics level, we just have two types of nodes (source and termination node) for each task.